griptape/drivers/vector/redis_vector_store_driver.py

```diff
@@ -0,0 +1,113 @@
+from typing import Optional, List
+from griptape import utils
+from griptape.drivers import BaseVectorStoreDriver
+import redis
+from attr import define, field
+import json
+from redis.commands.search.query import Query
+import numpy as np
+
+
+@define
+class RedisVectorStoreDriver(BaseVectorStoreDriver):
+    host: str = field(kw_only=True)
+    port: int = field(kw_only=True)
+    db: int = field(kw_only=True, default=0)
+    password: Optional[str] = field(default=None, kw_only=True)
+    client: redis.StrictRedis = field(init=False)
+    index: str = field(kw_only=True)
+
+    def __attrs_post_init__(self) -> None:
+        self.client = redis.StrictRedis(
```

Comment:
I think `StrictRedis` is an old alias for `Redis`. So, you can change it to `Redis`.

Reply:
Could you also initialize it in the property factory like we do with other clients?

Reply:
Done!